tests/integration/TEC/Tickets/Emails/Admin/__snapshots__/Emails_TabTest__it_should_match_stored_json_for_general_settings__1.php

```diff
@@ -17,9 +17,13 @@
     },
     "1": {
         "type": "html",
-        "html": "<h3>Sender Information<\\/h3>"
+        "html": "<div class=\\"tribe-common\\">\\n\\t<div  class=\\"tec-admin__upsell tec-admin__upsell-tec-tickets-wallet-plus\\" >\\n\\t\\t<div class=\\"tec-admin__upsell-content\\">\\n\\t\\t\\t<div class=\\"tec-admin__upsell-icon\\">\\n\\t<img\\n\\t\\tclass=\\"tec-admin__upsell-icon-image\\"\\n\\t\\tsrc=\\"http:\\/\\/wordpress.test\\/wp-content\\/plugins\\/the-events-calendar\\/common\\/src\\/resources\\/images\\/icons\\/circle-bolt.svg\\"\\n\\t\\talt=\\"The Events Calendar important notice icon\\"\\n\\t\\/>\\n<\\/div>\\t\\t\\t<div class=\\"tec-admin__upsell-text-wrap\\">\\n\\t\\t\\t\\t<span class=\\"tec-admin__upsell-text\\">\\n\\t\\t\\t\\t\\tGet additional ticketing flexibility including Apple Wallet and PDF tickets with \\t\\t\\t\\t<\\/span>\\n\\t\\t\\t\\t<a \\n\\t\\t\\t\\t\\t class=\\"tec-admin__upsell-link tec-admin__upsell-link--underlined\\"  \\n\\t\\t\\t\\t\\thref=\\"https:\\/\\/evnt.is\\/1bdz\\" \\n\\t\\t\\t\\t\\trel=\\"noopener noreferrer\\"\\n\\t\\t\\t\\t\\ttarget=\\"_blank\\"\\n\\t\\t\\t\\t>\\n\\t\\t\\t\\t\\tEvent Tickets Plus\\t\\t\\t\\t<\\/a>\\n\\t\\t\\t<\\/div>\\n\\t\\t<\\/div>\\n\\t<\\/div>\\n<\\/div>"
```

Comment:
Why is this snapshot being changed ?

Reply:
A test was bleeding into others - found and fixed, thanks!